### PR TITLE
New FF pages per PRs 4908, 4927, 4923

### PR DESCRIPTION
**Pull request by peiying2**
https://github.com/mozilla/bedrock/pull/4908
https://github.com/mozilla/bedrock/pull/4927
https://github.com/mozilla/bedrock/pull/4923

See if I am on the right track with comments. 

NOTE: 
- "Features" appears on Nav bar but also as a section name.  Not sure if this matters at all.
- "Download Firefox" appears on all pages but only shows up in private-browsing.lang.  It is in download_button.lang so do we remove it from here?
- "Firefox Privacy" appears on all pages but only shows up in focus.lang.  
- "Firefox" shows up in several files. Do we save it in the shared file or remove it all together?
- Blogs: this section is excluded for l10n. But for German, we are building the blog site. I have not checked the code to see whether German will be made available at a later time. 

NOTE#2 Ignore IH pages as I am far from done with those.

### CODE REVIEW

File: en-US/firefox/features/bookmarks.lang [hunk 1]
```diff
@@ -18,53 +20,30 @@ Better bookmarks
 Don’t agonize, lovers of bookmarking. Organize with Firefox.
 
 
-;Features
-Features
-
-
+# File means to arrange (that bookmark).
```
Comment (flodolo) [inline en-US/firefox/features/bookmarks.lang]: Maybe save instead of arrange?

File: en-US/firefox/features/bookmarks.lang [hunk 2]
```diff
@@ -18,53 +20,30 @@ Better bookmarks
...
 ;File that bookmark
 File that bookmark
 
 
+# Dial in means to access (the bookmarks).
```
Comment (flodolo) [inline en-US/firefox/features/bookmarks.lang]: I don't think it means that from the context, more like "show", "enable"?

File: en-US/firefox/features/bookmarks.lang [hunk 3]
```diff
@@ -18,53 +20,30 @@ Better bookmarks
...
 ;Get your faves all sorted with the bookmark star icon, which lets you add custom names and folders quickly. Then dial in your bookmarks toolbar to make sure you never lose sight of the links you love.
 Get your faves all sorted with the bookmark star icon, which lets you add custom names and folders quickly. Then dial in your bookmarks toolbar to make sure you never lose sight of the links you love.
 
 
+# Fly here means to have your bookmark go wherever you go.
 ;Fly with that bookmark
 Fly with that bookmark
 
 
+# On the fly means doing something quickly without giving it too much thought.
```
Comment (flodolo) [inline en-US/firefox/features/bookmarks.lang]: It sounds more "in mobility", "on the move" to me.

File: en-US/firefox/features/bookmarks.lang [hunk 4]
```diff
@@ -18,53 +20,30 @@ Better bookmarks
...
 ;File that bookmark
 File that bookmark
 
 
+# Dial in means to access (the bookmarks).
 ;Get your faves all sorted with the bookmark star icon, which lets you add custom names and folders quickly. Then dial in your bookmarks toolbar to make sure you never lose sight of the links you love.
 Get your faves all sorted with the bookmark star icon, which lets you add custom names and folders quickly. Then dial in your bookmarks toolbar to make sure you never lose sight of the links you love.
 
 
+# Fly here means to have your bookmark go wherever you go.
```
Comment (flodolo) [inline en-US/firefox/features/bookmarks.lang]: Also specify that English plays with the next "on the fly", but it can be ignored if not possible during translation

File: en-US/firefox/features/bookmarks.lang [hunk 5]
```diff
@@ -18,53 +20,30 @@ Better bookmarks
...
 ;Fly with that bookmark
 Fly with that bookmark
 
 
+# On the fly means doing something quickly without giving it too much thought.
 ;Take your favorites on the fly. Use Firefox Sync to access your bookmarks across all your devices, from desktop to mobile. Or try <a href="%(url)s">Pocket</a> to save any online article, or page and come back to later – even without internet.
 Take your favorites on the fly. Use Firefox Sync to access your bookmarks across all your devices, from desktop to mobile. Or try <a href="%(url)s">Pocket</a> to save any online article, or page and come back to later – even without internet.
 
 
+# Futz with means to busy with oneself.
```
Comment (flodolo) [inline en-US/firefox/features/bookmarks.lang]: I'm not sure the explanation helps me :-)

I find "To manipulate something without a clear sense of purpose". No clue how we can explain these to non native speakers…

File: en-US/firefox/features/fast.lang [hunk 6]
```diff
@@ -18,18 +16,17 @@ Our new, powerful multi-process platform handles all your tabs without slowing d
...
 
-;Features
-Features
-
-
 ;Use less memory
 Use less memory
 
 
+# A hog is something that claims or takes all of everything.
```
Comment (flodolo) [inline en-US/firefox/features/fast.lang]: I would add: here it references to a program that takes up all the resources and slow down the computer.

File: en-US/firefox/features/fast.lang [hunk 7]
```diff
@@ -46,29 +43,7 @@ Multi-tasking with multiple tabs just got easier. Firefox is now a multi-process
 Level-up browser gameplay
 
 
+# Native speed refers to operating speed for games written for platforms such as macOS or Windows.
+# Ping refers to responsive time. It is measured in milliseconds.
```
Comment (flodolo) [inline en-US/firefox/features/fast.lang]: Not sure the explanation for ping is needed?

File: en-US/firefox/features/index.lang [hunk 8]
```diff
@@ -22,10 +22,12 @@ Modern
 Your favorite add-ons and extensions
 
 
+# These are add-on app names. Product names remain in English. If needed, add purpose of apps in parathesis right after the product name.
 ;Adblock, Ublock, LastPass and thousands more.
 Adblock, Ublock, LastPass and thousands more.
 
 
+# This is a link.
```
Comment (flodolo) [inline en-US/firefox/features/index.lang]: to? I would rephrase all these as "Link to X"

File: en-US/firefox/features/index.lang [hunk 9]
```diff
@@ -146,10 +157,12 @@ Block ad trackers
 More powerful Private Browsing
 
 
+# Incognito and InPrivate are features offered by Chrome and IE respectively. Feature names should remain in English unless they are localized in your language.
```
Comment (flodolo) [inline en-US/firefox/features/index.lang]: Second part: "Check a localized version of each software to see how these features are translated, otherwise keep them in English".

File: en-US/firefox/features/password-manager.lang [hunk 10]
```diff
@@ -30,6 +28,7 @@ Password hero
 Forget the reset. Firefox Password Manager keeps all your passwords so you can log in automatically, or find saved passwords easily. For super security, give your computer a <a href="%(url)s">master password</a>.
 
 
+# Ninja is a skilled fighter trained in Japanese martial arts. Here it means a protector. Feel free to substitue with character known to local culture.
```
Comment (flodolo) [inline en-US/firefox/features/password-manager.lang]: I don't think it means protector here? I think ninja here means "master", "expert" or similar

File: en-US/firefox/features/private-browsing.lang [hunk 11]
```diff
@@ -2,14 +2,15 @@
 ## URL: https://www-dev.allizom.org/%LOCALE%/firefox/features/private-browsing
 
 
+# Used as HTML title of the page
 ;Private Browser with extra tracking protection
 Private Browser with extra tracking protection
 
 
 ;Download Firefox
 Download Firefox
```
Comment (flodolo) [inline en-US/firefox/features/private-browsing.lang]: We already have this from download_button, it should be safe to remove

File: en-US/firefox/hub/home.lang [hunk 12]
```diff
@@ -18,47 +14,15 @@ Firefox is now even better on Desktop, Windows, Mac, Linux, iOS and Android. See
 Firefox browser
 
 
-# like you’re meant to… fast, free and privately."
+# Alternative string for the following: Brows like you’re meant to… fast, free and privately.
```
Comment (flodolo) [inline en-US/firefox/hub/home.lang]: browse

File: en-US/firefox/shared.lang [hunk 13]
```diff
@@ -31,6 +35,49 @@ Features
...
+# This is a link.
+;Browse independently with Firefox
+Browse independently with Firefox
+
+
+;More performance
+More performance
+
+
+# Bring it can be translated as ""Let's go".
```
Comment (flodolo) [inline en-US/firefox/shared.lang]: Double "

File: en-US/mozorg/internet-health/index.lang [hunk 14]
```diff
@@ -103,10 +103,6 @@ The Internet was built on the promise that any one of us might create the next b
 Support Internet Openness
 
 
-;Net Neutrality
-Net Neutrality
```
Comment (flodolo) [inline en-US/mozorg/internet-health/index.lang]: Let's not move strings around for internet-health ;-)

File: en-US/firefox/shared.lang [hunk 15]
```diff
@@ -1,6 +1,10 @@
 ## NOTE: this is a shared file used across /firefox and /firefox/features pages
 
 
+;Firefox
+Firefox
```
Comment (flodolo) [inline en-US/firefox/shared.lang]: Let's not expose Firefox for translation

File: en-US/firefox/features/bookmarks.lang (hunk 5 above)
Comment (peiying2) [inline en-US/firefox/features/bookmarks.lang]: The word is an old expression. It means to busy with oneself aimlessly and probably not something meaningful. We need to avoid using those words in this context, :).

File: en-US/firefox/features/index.lang (hunk 8 above)
Comment (peiying2) [inline en-US/firefox/features/index.lang]: An example will be "Link to Independent page"?

Comment (flodolo) [inline en-US/firefox/features/index.lang]: Link to /firefox/independent, or link to URL if external, etc.

File: en-US/firefox/features/password-manager.lang (hunk 10 above)
Comment (peiying2) [inline en-US/firefox/features/password-manager.lang]: Ninja is characterized as power/skillful, speed, secretive... In the same section, Hero and Master are used.  I have not updated this yet.

File: en-US/firefox/features/fast.lang (hunk 7 above)
Comment (peiying2) [inline en-US/firefox/features/fast.lang]: If this is commonly understood, I can remove it.

File: en-US/mozorg/internet-health/index.lang (hunk 14 above)
Comment (peiying2) [inline en-US/mozorg/internet-health/index.lang]: This string appears in both the Index and decentralization files.
